modules/programs/qutebrowser.nix

```diff
@@ -131,7 +131,8 @@ in {
     };
 
     keyBindings = mkOption {
-      type = types.attrsOf (types.attrsOf types.str);
+      type = let coerce = concatStringsSep " ;; ";
+      in with types; attrsOf (attrsOf (coercedTo (listOf str) coerce str));
```

Comment:
```suggestion
      type = with types; attrsOf (attrsOf (separatedString " ;; "));
```

Reply:
Is there a reason to prefer one way over the other (readability or otherwise)? The first version of the type was suggested just before your review. https://github.com/nix-community/home-manager/pull/3322#discussion_r1028533628

Reply:
It's a trade-off between simplicity of types and brevity of definitions. I am partial to the former, but it is only a suggestion.

Reply:
I'll just go with your suggestion, it looks nicer to me as well.